cnf-certification-test/networking/netcommons/netcommons.go

```diff
@@ -150,8 +168,10 @@ func FilterIPListByIPVersion(ipList []string, aIPVersion IPVersion) []string {
 func FindRogueContainersDeclaringPorts(containers []*provider.Container, portsToTest map[int32]bool) []string {
 	var rogueContainers []string
 	for _, cut := range containers {
+		var portsAlreadyReportedRogue []int32
 		for _, port := range cut.Ports {
-			if portsToTest[port.ContainerPort] {
+			if portsToTest[port.ContainerPort] && !ExistsInInt32Slice(port.ContainerPort, portsAlreadyReportedRogue) {
```

Comment:
Do we really want this? Containers from different pods are allowed to open the same port number. I believe we should print a trace for each container so partners can fix several pods before running the TNF cert tool again.

Reply:
Let me look at this again, you may be right.

Reply:
Okay I removed this one.